tests/test_app/config/AltSeeds/NumbersAltSeed.php

```diff
@@ -18,7 +18,6 @@ public function run()
     {
         $data = [
             [
-                'id' => '1',
```

Comment:
It is a documented feature of those databases. It is possible to workaround those cases, but it requires specific code and special configuration for each of the drivers